### DISCUSSION TITLE
Wie komme ich an ein Crashlog ran?

### DISCUSSION THREAD

**Discussion by Conan179** [Freetz-NG/freetz-ng#263] [Allgemein]
Hi, ich hab heute abend ein neues Image gebaut, meine änderungen waren, ein git pull und hab das vsftp endfernt.
Jetzt Startet meine 6590 alle paar minuten neu.
Damit ich rausfinden kann warum sie das macht, ist meine frage wie komme ich an einen crashlog ran?
Das Image wo immer wieder neustartet ist:
FritzOS 07.25 rev87194 (22.03.2021 13:55:32)
Freetz ng-18229M-eca6d622b (09.04.2021 21:29:39)

Im punkt "AVM-logs" habe ich noch "hdl=18 aha           triggered before: 130.520 s(avg   0.000 s) maybe crashed" gefunden, bevor sie neugestartet ist.

**Comment by fda77** [comment]
Die Crash-Logs von AVM sollten auch noch nach einem Neustart vorhanden sein, falls ja sind sie unter avm-logs zu sehen.
Ich nuzte das Syslog Package und hab es so konfiguriert dass es zu einem anderen Gerät im Netzwerk weiterleitet. Dafür brauch man aber irgend einen Syslog-Server.
Die Meldung sieht für mich so aus dass aha (Smart Home) gecrasht ist und der AVM-Watchdog deshlab nach 130 sec einen Reboot ausgelöst hat. Das kann mal vorkommen wenn die AVM-Konfiguration durcheinander komm, zb durch Downgrade.

**Reply by Conan179** [reply]
Zuerst, endschuldigung, das ich das heir als issue gepostet hatte.
Ich hab Gestern abend noch die nandbank gewechselt, um wieder in das image rein zu kommen wo nicht neustartet, was aber auch lötzlich gemacht hat, ich hab auf die schnelle ein image gebaut, wo ich das aha raus gepatcht hab und gefasht, keine neustarts mehr.
Ich weis, ist ne Quick and Dirty lösung, smart home nutzte ich eh nicht zuhause.
Einmal habe ich es geschaft, über das freetz webinterface support daten zu erstellen, da ist ein crash.log drin, ich hänge sie an, evtl. erkennst du was drin, ich persönliche verstehe null davon.
[crash.log](https://github.com/Freetz-NG/freetz-ng/files/6289778/crash.log)

**Comment by fda77** [comment]
Ich seh da auch nur dass wieder Smarthome/AHA die Ursache ist
```
[Segmentation fault] AVM Home Automation([4858]) CRASHED at 0+0xb76ee351 (/lib//libavmcsock.so.2 at 00015351) accessing 00000003
```

und überall libavmcsock.so, irgendwas mit (Netzwerk-)Sockets

Da das Gerät keine Internetverbindung hat ist auch keine korrekte Uhrzeit:  `2000-06-08 05:51:06(1)` wodurch man nicht genau weiss ob das eine uraltes oder aktuelles Log ist
Vielleicht erkennen andere die sich da besser auskennen mehr

Es gab schon öfter Fälle dass die AVM-Config durcheinander kam und dann mache Dienste oder die ganze Box nicht mehr erreichbar war. Ob das durch Freetz begünstigt wird ist nicht klar. Da sie beide Paritionen die gleich config Teilen trifft es dann auf beide zu. Ein allgemeines Problem ist es nicht, ich hab schon lange Steckdosen+Heizkörperregler an der 7590

**Reply by Conan179** [reply]
Also, ich hatte jetzt versucht ne ältere Settings wiederherzustellen, ging nicht.
Um was anderes zu testen (mache ein neues Thema dafür auf) bin ich zurück auf die 7.20 fw gegangen und dort die letzten 7.20 settings wiederhergestellt, dann wieder auf 7.25 geupdatet, kommplet ohne remove patches, also mit allem, die Box läuft wieder ohne neustarts.

Danke für deine Hilfe.

**Reply by fda77** [reply]
Falls ein Remove Patch das verursachte muss du herausfinden welcher. Immer ein paar mehr aktivieren und flashen...
Ich hab auf der 7590 relativ viele remove-patches aktiviert, aha/smarthome funktioniert bei mir trotzdem

**Reply by Conan179** [reply]
Ich hatte nur 2 removes patchs drin gehabt, wobei ich nichtm ehr weis warum, das waren remove leftover und dtrace, die waren aber nicht der übeltäter, die waren schon die ganze zeit ausgewählt, warum der aha den watchdog getriggert hat, werde ich woll nie herausfinden, ist aber für mich nicht schlimm, jetzt läuft sie wieder.

**Reply by fda77** [reply]
Das hier ist bestimmt auch so was: https://github.com/Freetz-NG/freetz-ng/issues/225
Läuft bei dir PLCD eigentlich? Ich hab den entfernt

**Reply by Conan179** [reply]
Ähm, was ist PLCD?

**Reply by fda77** [reply]
Irgendwas mit PowerLine was ständig ins Netzwerk brodcastet. Ich hab so keine Geräte

**Reply by Conan179** [reply]
Achso du meinst mit plcd powerline, tschuldigung, dazu kann ich nihts sagen, hab auch keine powerline geräte in nutzung.

**Reply by fda77** [reply]
Ich kann nichts dafür wie AVM die binaries nennt

**Reply by Conan179** [reply]
Na, das meinte ich auch gar nicht. Du kennst den begriff plcd als powerline, ich nicht.
Dement sprechend, war mir nicht klar, was du meinst, war kein Vorwurf ;)

**Reply by fda77** [reply]
"d" am Ende ist oft für "daemon". Es heisst wohl offiziell nicht powerline sondern "powerlan" https://de.wikipedia.org/wiki/PowerLAN
